### ISSUE TITLE
Consider more semantic-neutral tag names

### ISSUE THREAD

**Issue by matklad** [jgm/djot#122]
This is super-weak nice to have, but we have, eg `code_block` and `emoji` tag names, which are specific to _one_ use case. Given the extensibility goal of djot, we might want to go for more general names, like `verbatim_block` and `subst`.

**Comment by jgm** [comment]
Agreed. I've been trying to think of a good name to use instead of `emoji`.
Not suer about `subst`.

**Comment by matklad** [comment]
`symbol` as another alternative. :thinking: actually, as `:thinking:` is an alternative way to write :thinking:, `alt` might also work.

**Comment by jgm** [comment]
I like `symbol`.

**Comment by dereuromark** [comment]
symbol is great, yes. 👍

**Comment by dereuromark** [comment]
Looking at the reference implementation: 
  1. emoji → symbol — ✅ Done                                                                                                    
  2. code_block → verbatim_block — ❌ Not done (and jgm seemed uncertain about it)       

So should we move forward with the 2nd part for now? Or close this as good enough?
@jgm How do we do this?

**Comment by jgm** [comment]
I'd be okay with `code_block` -> `verbatim_block`.
(my only reservation is that it's a change, which requires work across all implementations, and which may introduce bugs -- is it worth it?)
